shared/templates/sebool/ansible.template

```diff
@@ -4,34 +4,44 @@
 # complexity = low
 # disruption = low
 
+{{% if product in ['ol8', 'ol9', 'rhel8', 'rhel9'] %}}
+{{% set PACKAGE_NAME = "python3-libsemanage" %}}
+{{% elif product == "sle15" %}}
+{{% set PACKAGE_NAME = "policycoreutils" %}}
+{{% else %}}
+{{% set PACKAGE_NAME = "libsemanage-python" %}}
+{{% endif %}}
+
+- name: Ensure {{{PACKAGE_NAME}}} installed
+  package:
+    name: "{{{ PACKAGE_NAME}}}"
+    state: present
+
+{{% if product == "sle15" %}}
+- name: Ensure list of additional packages are installed on SLE 15
```

Comment:
Since we are touching this Ansible file, could you also update all tasks names in this Playbook in alignment to the Project Style Guide, please?
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible